.gitignore

```diff
@@ -1,16 +1,16 @@
-.coverage
+.coverage/
```

Comment:
Is there a semantic difference or did you just want to indicate `.coverage` is a directory?

Reply:
yeah just to indicate